jina/hubble/helper.py

```diff
@@ -250,3 +252,13 @@ def upload_file(
     response = getattr(requests, method)(url, data=data, headers=headers, stream=stream)
 
     return response
+
+
+def install_requirements(requirements_file: 'Path'):
+    """Install modules included in requirments file
+
+    :param requirements_file: the requirements.txt file
+    """
+    subprocess.check_call(
```

Comment:
isn't there a `python pip client`?

Reply:
@JoanFM This is the officially recommended way https://pip.pypa.io/en/latest/user_guide/#using-pip-from-your-program. Furthermore since pip v10, pip has some breaking changes.